InstallDebian.md

```diff
@@ -0,0 +1,83 @@
+# TDS Foreign data wrapper
+
+* **Author:** Geoff Montee
+* **Name:** tds_fdw
+* **File:** tds_fdw/InstallDebian.md
+
+## Installing on Debian
+
+This document will show how to install tds_fdw on Debian 10. Other Debian distributions should be similar.
+
+### Install FreeTDS
+
+The TDS foreign data wrapper requires a library that implements the DB-Library interface,
+such as [FreeTDS](http://www.freetds.org).
+
+```bash
+sudo apt-get install libsybdb5 freetds-dev freetds-common
+```
+
+### Install PostgreSQL
+
+If you need to install PostgreSQL, do so by following the [apt installation directions](https://wiki.postgresql.org/wiki/Apt). For example, to install PostgreSQL 11 on Debian:
+
+```bash
+sudo bash -c "echo \"deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -c -s)-pgdg main\" > /etc/apt/sources.list.d/pgdg.list"
+wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
```

Comment:
On Docker images (and maybe on really minimal Debian systems) this requires a couple of extra packages, so we better add this line above:
```
sudo apt-get install gnupg wget
```